### PR TITLE
(Fix) Validate whitelist min/max

### LINKED ISSUE #578: (Fix) add validation for Whitelist->min/max value
### Problem
Min, Max value can be negative
Min can be more or equals than Max 

If you are reporting a problem with Token Wizard, please include the following information:

### Which network did you use? (Mainnet, Kovan, Rinkeby, etc.)

*n/a*

### If you were able to create it, what is the URL of your crowdsale?

*n/a*

### Do you have screenshots showing the problem?

whitelistminmax

### Do you see errors in the dev console? If yes, please include a screenshot

To open the dev console in Google Chrome, press F12, or go to `View -> Developer -> Developer Tools`, and open the Console tab

If you see errors, please right click on them and "Save as..". Zip saved file and attach it to the Issue.

*No*

---

### PR DESCRIPTION
**Pull request by fernandomg**
Closes #578 

This PR depends on changes made in #767.

- Validates min/max in whitelist block
- Validates min/max for bulk upload
- Changes where made for `StepThree` and `Manage` components

Commits made for this PR:
 - https://github.com/poanetwork/token-wizard/pull/778/commits/e93165cb5d9a38b645373b509649ac3bd6fa9a0b
 - https://github.com/poanetwork/token-wizard/pull/778/commits/f0091391713d22243fbf0e3381d32c9dc13cfc7f
 - https://github.com/poanetwork/token-wizard/pull/778/commits/54377281c53205309b7ccf720e67ac1ccb1b28c2
 - https://github.com/poanetwork/token-wizard/pull/778/commits/bab0e896cc38c78dffae88e21ccb7d53639d128d

### CODE REVIEW

**Comment by vbaranov** [comment]
@fernandomg please resolve conflicts

**Comment by fernandomg** [comment]
@vbaranov done

File: src/utils/processWhitelist.spec.js [hunk 1]
```diff
@@ -83,7 +83,7 @@ describe('processWhitelist function', () => {
     const cb = jest.fn()
 
     // When
-    const { called } = processWhitelist(rows, cb)
+    const { called } = processWhitelist({ rows, decimals: 3 }, cb)
```
Comment (fvictorio) [inline src/utils/processWhitelist.spec.js]: Could you add some test cases for ignoring invalid decimals, and invalid min/max combinations?

File: src/utils/validations.js [hunk 2]
```diff
@@ -20,6 +20,28 @@ export const validateTicker = (value) => {
...
 
+export const validateWhitelistMin = ({ min, max, decimals }) => {
+  const listOfErrors = composeValidators(
+    isRequired(),
+    isNonNegative(),
+    isDecimalPlacesNotGreaterThan(`Decimals should not exceed ${decimals} places`)(decimals),
+    isLessOrEqualThan('Should be less or equal than max')(max)
+  )(min)
+
+  return listOfErrors ? listOfErrors.shift() : ''
```
Comment (fvictorio) [inline src/utils/validations.js]: Shouldn't this return `undefined` when there are no errors?

File: src/utils/processWhitelist.spec.js (hunk 1 above)
Comment (fernandomg) [reply src/utils/processWhitelist.spec.js]: done

File: src/utils/validations.js (hunk 2 above)
Comment (fernandomg) [reply src/utils/validations.js]: yes, fixed

**Pull request merged**